src/services/shell.cpp

```diff
@@ -370,8 +371,14 @@ void Shell::install()
 
     installRequiredPackages();
 
-    const auto& repos = Repos(m_cluster->getHeadnode().getOS());
-    repos.configureRepositories();
+    auto repos = m_cluster->getRepoManager();
+    repos.loadFiles();
+    repos.enableMultiple({ "cloyster-beegfs", "cloyster-elrepo",
+        "cloyster-epel", "cloyster-openhpc",
```

Comment:
What about the names here? We have mentioned to use `PRODUCT_NAME` instead or just use the default distribution names.